MAUI/TreeView/working-with-treeview.md

```diff
@@ -0,0 +1,135 @@
+---
+layout: post
+title: Working with TreeView in .NET MAUI TreeView control | Syncfusion
+description: Learn here all about Working with TreeView support in Syncfusion .NET MAUI TreeView (SfTreeView) control and more.
+platform: .NET MAUI
+control: SfTreeView
+documentation: ug
+---
+
+# Working with TreeView in .NET MAUI TreeView (SfTreeView)
+
+## Interacting with TreeView Items
+
+### Loaded event
+
+The `Loaded` event is raised when the TreeView is loading in view for the first time.
+
+{% tabs %}
+{% highlight c# %}
+treeView.Loaded += TreeView_Loaded;
+
+private void TreeView_Loaded(object sender, TreeViewLoadedEventArgs e)
+{
+   DisplayAlert("Message", "TreeView is Loaded", "Done");
+}
+{% endhighlight %}
+{% endtabs %}
+
+The `Loaded` event is used for the following use case:
+
+* To scroll the desired item by using the `BringIntoView`.
+
+### Tapped event
+
+The `ItemTapped` event will be triggered whenever tapping the item.  `ItemTappedEventArgs` has the following members which provides the information for `ItemTapped` event:

```

Comment:
The `ItemTapped` event will be triggered whenever tapping the item.  `ItemTappedEventArgs` has the following members which provide the information for the `ItemTapped` event: